src/main/java/com/limechain/sync/warpsync/SyncedState.java

```diff
@@ -57,7 +56,10 @@
 @Log
 public class SyncedState {
     private static final SyncedState INSTANCE = new SyncedState();
-    protected SyncedState() {}
+
+    protected SyncedState() {
+    }
```

Comment:
this can be an annotation - `@NoArgsConstructor` with the visibility argument set to protected